lib/mix/lib/mix/tasks/xref.ex

```diff
@@ -159,7 +166,7 @@ defmodule Mix.Tasks.Xref do
     ["  ", file, ?:, Integer.to_string(line), ?\n]
   end
 
-  ## Helpers
+  ## Unreachable helpers
```

Comment:
So this means that the following are helpers for the "unreachable" feature? I found this comment not super clear 😕, also because the next section is called "Callers". ""Callers" helpers"? :)